app/controllers/items_controller.rb

```diff
@@ -7,11 +8,14 @@ def new
   end
 
   def create
-    @new_item = Item.new(safe_params)
+    @new_item = current_user.items.build(safe_params)
     if @new_item.save
+      flash.now[:success] = "You added a #{@new_item.model} for: #{current_user.email}."
       redirect_to @new_item
     else
+      flash.now[:error] = "We were unable to add a new item - please try again"
```

Comment:
might be helpful to show the use the errors here to enable them to fix whatever went wrong